### ISSUE TITLE
TODO: Add support for Intercom Engage?

### ISSUE THREAD

**Issue by jaswrks** [websharks/woocommerce-intercom-pro#13]
Referencing: https://github.com/websharks/woocommerce-intercom-pro/blob/160909.7530/src/includes/classes/Utils/JsSnippet.php#L86

**Comment by jaswrks** [comment]
@raamdev It looks like Engage is already working with the existing code. Can you help me understand what you had in mind when you wrote this TODO item please?

**Comment by jaswrks** [comment]
Oh. I see that it's Acquire that we added, not Engage. I haven't tested Engage at all yet.

**Comment by raamdev** [comment]
@jaswsinc Right. I haven't dug into how Engage might be different or might require additional integration. This TODO was just a reminder that we need to research that feature to see how the WooCommerce Intercom plugin might add support for it.
